### PR TITLE
Make the GitGitGadget home page less dreary

### PR DESCRIPTION
**Pull request by dscho**
... and more informative.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,23 +1,79 @@
...
+			</p><p>
+				A few tutorials out there try to help with this process (e.g. <a href="https://github.com/git-for-windows/git/blob/master/CONTRIBUTING.md">Git for Windows' detailed instructions how to contribute patches to the Git project</a>). GitGitGadget tries a different approach: allow contributing patches to the Git project itself by opening a Pull Request at https://github.com/gitgitgadget/git and let GitGitGadget prepare and send the corresponding mails.
+			</p>
+		</div>
+		
+		<h2>How can you use GitGitGadget?</h2>
+		<div class="block">
+			<p>
+				So you cloned https://github.com/git/git and implemented a bug fix or a new feature? And you already pushed it to your own fork? Good, now is the time to direct your web browser to https://github.com/gitgitgadget/git and to open a Pull Request. Please make sure to use a descriptive Pull Request title and description; GitGitGadget will use these as the cover letter.
```
Comment (derrickstolee) [inline index.html]: https://github.com/git/git could be a hyperlink.

File: index.html [hunk 2]
```diff
@@ -1,23 +1,79 @@
...
+			Contributing git.git patches via GitHub PRs<br />
+			<tiny>... one iteration at a time... </tiny>
+		</div><hr />
 
-		<p>
-		This project intends to help submit patch series to the Git mailing list. The process is not quite as painless for the contributor as opening Pull Requests, but at least it is much less painful than having to perform all the steps manually (see e.g. <a href="https://github.com/git-for-windows/git/blob/master/CONTRIBUTING.md">Git for Windows' detailed instructions how to contribute patches to the Git project</a>).
-		</p>
+		<div class="block">
+			<p>
+				Unlike most open source projects, Git itself does not accept code contributions via Pull Requests. Instead, patches are <a href="https://github.com/git/git/blob/master/Documentation/SubmittingPatches">submitted to the Git mailing list</a> for review and will be applied manually by the Git maintainer.
```
Comment (derrickstolee) [inline index.html]: It's difficult to review this one-line-per-paragraph text.

File: index.html [hunk 3]
```diff
@@ -1,23 +1,79 @@
...
+				Once everything is ready to go, add a comment to that Pull Request saying <tt>/submit</tt>. This will trigger GitGitGadget (you can see the progress via the Check called "GitGitGadget PR Handler"): it will wrap your Pull Request into a nice bundle of mails in the format expected on the Git mailing list.
+			</p>
+		</div>
 
-		<p>
-		GitGitGadget's idea is for users to open a Pull Request at <a href="https://github.com/gitgitgadget/git">https://github.com/gitgitgadget/git</a> with a good description of their patch series. Then, the command <tt>/submit</tt>, issued via a comment on said PR will tell GitGitGadget to send the patches as mail thread to <a href="mailto:mailto:git@vger.kernel.org">the Git mailing list</a>, with the Pull Request's description as cover letter.
+		<h2>What happens after GitGitGadget sent the mails?</h2>
+		<div class="block">
+			<p>
+				The patches will be reviewed by volunteers (be gentle...) and you will most likely receive helpful comments. It is a good idea to wait at least one day (time zones, you know?) for reviewer comments to trickle in before sending another iteration of your patch series (if needed).
```
Comment (derrickstolee) [inline index.html]: The paranthetical statements here are well-meaning, but a bit casual. Consider rewriting. For instance

> It is a good idea to wait at least one day (time zones, you know?) for reviewer comments...

could be 

> The Git developer community is globally distributed, so please wait a day or two for reviewer comments...

File: index.html [hunk 4]
```diff
@@ -1,23 +1,79 @@
...
-		</p>
+		<div class="block">
+			<p>
+				Unlike most open source projects, Git itself does not accept code contributions via Pull Requests. Instead, patches are <a href="https://github.com/git/git/blob/master/Documentation/SubmittingPatches">submitted to the Git mailing list</a> for review and will be applied manually by the Git maintainer.
+			</p><p>
+				Apart from lacking the convenience of a web interface, this process also puts considerable demands on the code contributions: the mails are expected to be plain text only (no HTML!), for example, and the diffs embedded in the mails must apply cleanly (no whitespace changes!), among other things.
+			</p><p>
+				A few tutorials out there try to help with this process (e.g. <a href="https://github.com/git-for-windows/git/blob/master/CONTRIBUTING.md">Git for Windows' detailed instructions how to contribute patches to the Git project</a>). GitGitGadget tries a different approach: allow contributing patches to the Git project itself by opening a Pull Request at https://github.com/gitgitgadget/git and let GitGitGadget prepare and send the corresponding mails.
```
Comment (derrickstolee) [inline index.html]: You may want to more forcefully recommend reading this and the contributor guide before submitting a series, as that will help them know that they still need to fit style expectations, good commit messages, small commits, etc.

**Review by derrickstolee — DISMISSED** [review]
Overall looking good!

File: index.html (hunk 1 above)
Comment (dscho) [reply index.html]: Makes sense.

File: index.html (hunk 2 above)
Comment (dscho) [reply index.html]: I wrapped at the end-of-sentence markers.

File: index.html (hunk 3 above)
Comment (dscho) [reply index.html]: Done. Any more suggestions like this? (I'm a non-native speaker, and it shows...)

File: index.html (hunk 4 above)
Comment (dscho) [reply index.html]: I added a sentence in the section that talks about `/submit`.

**Comment by dscho** [comment]
@derrickstolee any outstanding issues?

**Comment by dscho** [comment]
> no time to re-check things

No worries. I'll merge, and we can always address issues later.

**Pull request merged**